Jenkinsfile

```diff
@@ -76,6 +79,15 @@ pipeline {
                 runBuild()
             }
         }
+        stage('Build Android on Java 8') {
```

Comment:
Would it make sense to move it near the `Tests Android Sample App` stage. Seems more like a safe guard than something we should verify early. At least it fails more often on the `Static Analysis` task on my PRs(🤦), so I'd prefer to postpone this test for later.

Reply:
Yes, that makes sense. I'll move it to the end.